runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -90,4 +90,4 @@ test {
 // Generates :runners:google-cloud-dataflow-java:runQuickstartJavaDataflow
 def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
 def gcsBucket = project.findProperty('gcsBucket') ?: 'temp-storage-for-release-validation-tests/quickstart'
-createJavaQuickstartValidationTask(name: 'Dataflow', gcpProject: gcpProject, gcsBucket: gcsBucket)
+createJavaQuickstartValidationTask(name: 'Dataflow', gcpProject: gcpProject, gcsBucket: gcsBucket, additionalArgs: ["diskSizeGb": "25"])
```

Comment:
Will the quickstart docs be updated to include this? We want the test to be as identical as possible to the quickstart docs. https://beam.apache.org/get-started/quickstart-java/